GeometryCore/src/Polygon.cpp

```diff
@@ -58,13 +70,55 @@ namespace geom_utils
     // Calculate and return convex hull for the polygon
     Polygon Polygon::convexHull() const 
     { 
-        return {}; 
+        //if (points.size() < 3) return; //can't bild a polygon from 2 points or less //idk how to do it
```

Comment:
Maybe just return `*this`?

Reply:
Yes, it works. thank you)

Reply:
Even for 3 points there is no sense to calculate convex hull. So, in fact, condition should be `if (points.size() <= 3)`